src/common/settings.js

```diff
@@ -1,35 +1,37 @@
 'use strict';
 
 const fs = require('fs');
+let deepmerge = require('deepmerge').default;
```

Comment:
As mentioned at https://github.com/mattermost/desktop/pull/586#discussion_r142157918, `const deepmerge = require('deepmerge')` is enough.

Reply:
@yuya-oc this form `const deepmerge = require('deepmerge')` failed when building. Did you try it and worked for you for both cases building and running tests?

Reply:
@dmeza Thanks for feedback. Somehow deepmerge becomes `object` from `function` in webpack. So error occurs at run time.
I'm not sure whether it's bug or spec of webpack though, let's keep this for now.